htmlstuff.php

```diff
@@ -14,8 +14,10 @@ function pageheader() {
 
 function datadisclosure() {
 	global $userdata;
-	echo('This data will be saved with your poll results. If there is too much personal information included, please contact luke-jr to improve the filtering <em>before</em> filling out the poll.<br>');
-	echo('Note that if you have not completed KYC with Coinbase, your results will be saved but ignored until you complete KYC <em>and</em> resubmit your poll results.<br>');
+	echo('<div id="welcome">');
+	echo('<h1>Hello '.$userdata['coinbase_userdata']['data']['name'].'</h1>');
```

Comment:
This isn't really part of the disclosure. I suggest moving the <div.../h1> part to outside the functions (just in coinbase.php for now).